## About Me 

<img class="profile-picture" src="profile.jpg" style="float: right;" align="right" width="400">

I am a postdoctoral researcher at The [Broad Institute](https://www.broadinstitute.org/) of MIT and Harvard co-advised by [Aviv Regev](https://www.broadinstitute.org/bios/aviv-regev) (now at [Genentech](https://www.gene.com/scientists/our-scientists/aviv-regev)) and [Fei Chen](https://www.insitubiology.org/people). 

Tissues are composed of cell types that each respond differently to external signals. Yet most tools used to study cell signaling are not cell type-specific, while single-cell tools do not measure proteins that actually encode and process signals. I aim to build single-cell proteogenomic tools that integrate direct measurements of dynamic protein activities across cell types, to enable a mechanistic and interpretable view of signaling in tissues. I am applying these tools to study the governing principles of how cell types integrate external signals, and how tissues respond drugs. 

I received my Ph. D. in [Systems Biology](https://ssqbiophd.hms.harvard.edu/) from Harvard University as a Paul and Daisy [Soros Fellow](https://www.pdsoros.org/meet-the-fellows/hattie-chung) advised by [Roy Kishony](https://kishony.technion.ac.il/) (now at Technionâ€“Israel Institute of Technology, where I spent 3 summers). My thesis was on evolutionary dynamics, in the context of how pathogens evolve during respiratory infections. 

I received my S.B. in [Biological Engineering](https://be.mit.edu/) from MIT with a minor in applied international studies.

hchung [AT] broadinstitute.org \| [Google Scholar](https://scholar.google.com/citations?user=kN3O3xsAAAAJ&hl=en) \| [Twitter](https://twitter.com/hattaca)

## Key Publications
<sub><sup>\* equal contribution, ^co-corresponding author</sup></sub>

1. Joint single-cell measurements of nuclear proteins and RNA in vivo. 
**Chung H^**, Parkhurst CN, Magee EM, Phillips D, Habibi E, Chen F, Yeung BZ, Waldman J, Artis D, Regev A^. ***Nature Methods*** 2021 [[link](https://www.nature.com/articles/s41592-021-01278-1)] [[pdf](pdfs/inCITEseq.pdf)] [[feature](https://www.nature.com/articles/s41592-022-01392-8)]. 
2. Rapid expansion and extinction of antibiotic resistance mutations during treatment of acute bacterial respiratory infections. 
**Chung H**, Merakou C, Schaefers MM, Flett KB, Martini S, Lu R, Blumenthal JA, Webster SS, Cross AR, Al Ahmar R, Halpin E, Anderson M, Moore NS, Snesrud EC, Yu HD, Goldberg JB, O'Toole GA, McGann P, Stam JA, Hinkle M, McAdam AJ, Kishony R, Priebe GP. ***Nature Communications*** 2022 [[link](https://www.nature.com/articles/s41467-022-28188-w)] [[pdf](pdfs/acute.pdf)] [[news](https://answers.childrenshospital.org/genomic-surveillance-bacteria/)]
3. Single-cell meta-analysis of SARS-CoV-2 entry genes across tissues and demographics. 
Muus C\*, Luecken M\*, Eraslan G\*, Waghray A\*, Heimberg G\*, Sikkema L\*, Kobayashi Y\*, Vaishnav ED\*, Subramanian A\*, Smillie CS\*, Jagadeesh K\*, Duong ET\*, Fiskin E\*, Torlai Triglia E\*, Ansari M\*, Cai P\*, Lin B\*, Buchanan J\*, Chen S\*, Shu J\*, Haber AL\*, **Chung H\***, Montoro DT\*, ... , NHLBI LungMAP Consortium. ***Nature Medicine*** 2021. *This was a large team effort at the start of the COVID-19 pandemic*. [[link](https://www.nature.com/articles/s41591-020-01227-z)]
4. A multi-view generative model for small molecule representation improves prediction tasks. 
Yin J\*, **Chung H\*^**, Regev A^. ***NeurIPS Learning Meaningful Representations of Life workshop*** 2020 [[link](https://drive.google.com/file/d/1LlHTQ307MGxHsJbRDIJrEGPLRCBvbk0P/preview)] [[pdf](pdfs/multiviewVAE.pdf)] [[selected talk](https://slideslive.com/38942737/a-multiview-generative-model-for-molecular-representation-improves-prediction-tasks)]
5. Global and local selection acting on the pathogen Stenotrophomonas maltophilia in the human lung. 
**Chung H**, Lieberman TD, Vargas SO, Flett KB, McAdam AJ, Priebe GP, Kishony R. ***Nature Communications*** 2017. [[link](https://www.nature.com/articles/ncomms14078)] [[pdf](pdfs/steno_lung.pdf)]

### Other Publications
6. The microbiota regulate neuronal function and fear extinction learning. 
Chu C, Murdock MH, Jing D, Won TH, **Chung H**, Kressel AM, Tsaava T, Addorisio ME, Putzel GG, Zhou L, Bessman NJ, Yang R, Moriyama S, Parkhurst CN, Li A, Meyer HC, Teng F, Chavan SS, Tracey KJ, Regev A, Schroeder FC, Liston C, Artis D. ***Nature*** 2019. [[link](https://www.nature.com/articles/s41586-019-1644-y)] [[news](https://www.nature.com/articles/d41586-019-03114-1)]
7. RNA structural determinants of optimal codons revealed by MAGE-Seq. 
Kelsic ED\*, **Chung H\***, Cohen N, Park J, Wang HH, Kishony R. ***Cell Systems*** 2016. [[link](https://www.sciencedirect.com/science/article/pii/S2405471216303684)] [[pdf](pdfs/codon_bias.pdf)] [[highlight](https://www.sciencedirect.com/science/article/pii/S2405471217300042)]
8. Inexpensive Multiplexed Library Preparation for Megabase-Sized Genomes. 
Baym M\*, Kryazhimskiy S\*, Lieberman TD\*, **Chung H\***, Desai MM, Kishony R. ***PLoS ONE*** 2015. [[link](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0128036)]
9. Isocost lines describe the cellular economy of genetic circuits. 
Gyorgy A, Jimenez JI, Yazbek J, Huang HH, **Chung H**, Weiss R, Del Vecchio D. ***Biophysical Journal*** 2015. [[link](https://www.sciencedirect.com/science/article/pii/S0006349515006177)]
10. Lipid-derived nanoparticles for immunostimulatory RNA adjuvant delivery. 
Nguyen DN, Mahon KP, Chikh G, Kim P, **Chung H**, Vicari AP, Love KT, Goldberg M, Chen S, Krieg AM, Chen J, Langer R. Anderson DG. ***Proceedings of the National Academy of Sciences*** 2012. [[link](https://www.pnas.org/doi/abs/10.1073/pnas.1121423109)]

## Recent News and Events
* [June 2022] Invited to teach inCITE-seq at a hands-on workshop on [Single cell profiling and analysis in neuroscience](http://cajal-training.org/on-site/scpan/), hosted by the Cajal Advanced Neuroscience Training Programme and the Bordeaux School of Neuroscience. 
* [February 2022] Invited talk @ MIT Department of Biological Engineering. 
* [February 2022] Interview in a Nature Methods Technology Feature on [How single-cell multi-omics builds relationships](https://www.nature.com/articles/s41592-022-01392-8). 
* [November 2021] Invited talk @ Genentech, Cell and Tissue Genomics division. 
* [October 2021] Invited talk @ UVA Symposium for Emerging Scholars in Genome Sciences
* [August 2021] Invited talk @ Single Cell Proteomics Conference. Recording [here](https://www.youtube.com/watch?v=gvAb5Q0abYo).
* [July 2021] Talk @ Cell Circuits & Epigenomics Seminar, Broad Institute. 
* [March 2021] Invited talk @ [Single Cell Genomics day](https://satijalab.org/scgd21/).


### Misc
Between my Ph. D. and postdoc (2016-2018), I worked with Andrew C. Weber, the former Assistant Secretary of Defense for Nuclear, Chemical & Biological Defense Programs. I co-authored [a white paper](https://www.belfercenter.org/publication/north-koreas-biological-weapons-program-known-and-unknown) on North Korea's Bioweapons program, which was featured in [The Washington Post](https://www.washingtonpost.com/world/national-security/microbes-by-the-ton-officials-see-weapons-threat-as-north-korea-gains-biotech-expertise/2017/12/10/9b9d5f9e-d5f0-11e7-95bf-df7c19270879_story.html), [Vox](https://lieu.house.gov/media-center/in-the-news/vox-here-s-what-war-north-korea-would-look), and [FiveThirtyEight](https://fivethirtyeight.com/features/north-koreas-nukes-may-not-be-its-biggest-threat/). I was an Emerging Leader in Biosecurity [Fellow](https://www.centerforhealthsecurity.org/our-work/emergingbioleaders/class-of-2018.html) through the Center for Health Security at Johns Hopkins. 

I enjoy cooking, piano, ballet, languages, reading, and traveling. 

Website design from [this repo](https://github.com/ankitsultana/researcher)
